frontend/src/pages/modelRegistrySettings/ModelRegistryTableRowStatus.tsx

```diff
@@ -118,8 +117,24 @@ export const ModelRegistryTableRowStatus: React.FC<ModelRegistryTableRowStatusPr
     }
   }
 
+  const isClickable =
```

Comment:
I think you can have this logic based on whether there's a popover instead of based on status.
```ts
const isClickable = popoverTitle && popoverMessages.length;
```